Example/Swift Example/View Controllers/UserAuthenticationViewController.swift

```diff
@@ -44,6 +48,23 @@ class UserAuthenticationViewController: UITableViewController {
     
     // MARK: - Lifecycle
     
+    override init(style: UITableViewStyle) {
+        super.init(style: style)
+        self.dateFormatter.locale = NSLocale.currentLocale()
+        self.dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
+        self.dateFormatter.timeZone = NSTimeZone(forSecondsFromGMT: 0)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        self.dateFormatter.locale = NSLocale.currentLocale()
```

Comment:
Since practically nothing will use the dateFormatter before the view is loaded, you can put these initialization logic in `viewDidLoad`